specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/computeRPCommon.json

```diff
@@ -1913,6 +1913,59 @@
         }
       },
       "description": "Specifies additional settings to be applied when patch mode AutomaticByPlatform is selected in Linux patch settings."
+    },
+    "AttachDetachDataDisksRequest": {
+      "properties": {
+        "dataDisksToAttach": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/DataDisksToAttach"
+          },
+          "description": "The list of managed data disks to be attached."
+        },
+        "dataDisksToDetach": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/DataDisksToDetach"
+          },
+          "description": "The list of managed data disks to be detached."
+        }
+      },
+      "description": "Specifies the input for attaching and detaching a list of managed data disks."
+    },
+    "DataDisksToAttach": {
+      "properties": {
+        "diskId": {
+          "type": "string",
+          "description": "ID of the managed data disk."
```

Comment:
is this an arm id? If yes, please annotate it with [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details)

Reply:
No this is not treated as Arm-id.  It's just id of any managed disk. cross checking what benefit do we get with arm-id . Currently manageddisk parameters doesn't consider this as ARM Id